phantazm-commons/src/main/java/com/github/phantazmnetwork/commons/vector/Vec3D.java

```diff
@@ -0,0 +1,74 @@
+package com.github.phantazmnetwork.commons.vector;
+
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Represents a 3-dimensional double-precision floating point vector.
+ */
+public interface Vec3D extends Comparable<Vec3D> {
```

Comment:
is fuzzy equals no longer a good metric?

Reply:
i removed it because it wasn't being used anywhere, though it might be useful in the future? idk

Reply:
isn't comparing doubles with fuzzy necessary?

Reply:
if you shift doubles by some small epsilon value, you get the same effect without having to use an explicit fuzzy comparison method (you probably noticed me doing this in a few places)